### PR TITLE
Add params page model

### PR DESCRIPTION
**Pull request by VkoHov**
- Implement paramsService
- Implement paramsModel
- Integrate onCurveInterpolationChange, onColorIndicatorChange and onActivePointChange to paramsModel

### CODE REVIEW

File: aim/web/ui_v2/src/services/models/params/paramsAppModel.ts [hunk 1]
```diff
@@ -0,0 +1,121 @@
+import paramsService from 'services/api/params/paramsService';
+import { IActivePoint } from 'types/utils/d3/drawHoverAttributes';
+import { CurveEnum } from 'utils/d3';
+import createModel from '../model';
+
+const model = createModel<Partial<any>>({});
+
+function getConfig() {
+  return {
+    refs: {
```
Comment (roubkar) [inline aim/web/ui_v2/src/services/models/params/paramsAppModel.ts:10]: let's remove `refs` object from the app config state and keep it one level up

**Review by roubkar — APPROVED** [review]

**Comment by roubkar** [comment]
@VkoHov the restructuring of config state for removing refs object can be done in a separate PR, the rest looks good!

**Review by KaroMourad — APPROVED** [review]
lgtm

**Review by arsengit — APPROVED** [review]
lgtm

**Pull request merged**